app/models/region.rb

```diff
@@ -0,0 +1,36 @@
+class Region < ApplicationRecord
+  extend FriendlyId
+  friendly_id :name, use: :slugged
+  enum level: {
+    root: 0,
+    organization: 10,
+    state: 20,
+    facility_group: 30,
+    facility: 40
+  }
```

Comment:
Is the idea that this enum structure allows us enough room to insert other region levels into the Simple codebase to support, say, a new country's requirements? (eg. subdistricts in bangladesh)

Reply:
Yup, exactly.

Reply:
And I realized that these should be strings in the database, not integers, so that each country's database can use whatever names for these they want.

Reply:
Making these integers feels... wrong? The names are arbitrary; it's the hierarchy links that matter.

Even in India, different areas have different structures.